src/agora/dozf.cpp

```diff
@@ -82,6 +82,10 @@ void DoZF::compute_precoder(const arma::cx_fmat& mat_csi,
             mat_dl_zf = mat_ul_zf * arma::inv(mat_calib);
         } else
             mat_dl_zf = mat_ul_zf;
+	// We should scaling the beamforming matrix
+	// So the IFFT output can scaled with OFDM_CA_NUM
+	// See Argos paper (Mobicom 2012) Sec. 3.4
+	mat_dl_zf /= abs(mat_dl_zf).max();
```

Comment:
These lines should be aligned with line 84. There are also some typos in the comment.

Reply:
fixed!